vpc/service/e2e_test.go

```diff
@@ -0,0 +1,354 @@
+package service
```

Comment:
If this test gets too big, it might be worth splitting out like we do with the `standalone` tests and out of test-local.

I think this would be good because the tests only "happen" to run on buildkite under the standalone test, because docker happens to be present. Otherwise they are silently skipped.

I think having a dedicated test target can help separate failures (makes it obvious if the failure is from vpc e2e vs standalone stuff) and makes it so we don't accidentally skip the vpc e2e tests from being run in the future via other refactors.

Reply:
That's a good idea. Let me figure out how to do it. The test will surely become big as I add more tests.